### ISSUE TITLE
[Bug]: Mushroom's theming is overridden by HA's built-in one starting with 2022.11

### ISSUE THREAD

**Issue by BVollmerhaus** [piitaya/lovelace-mushroom#813]
### Current Behavior

Home Assistant seemingly got a slight redesign with version `2022.11`, which adds borders to all built-in cards and increases their radius. After updating HA as well as Mushroom + [Mushroom Themes](https://github.com/piitaya/lovelace-mushroom-themes) to their latest versions (`2.3.0` and `0.0.6`, respectively), this styling is now also applied to Mushroom-specific cards, meaning that:

* Mushroom cards appear to be affected by the redesign of HA's built-in cards and now display a border as well.
* The latest Mushroom Themes version no longer themes the built-in cards to be in line with Mushroom's styling.

After updating, all cards – including Mushroom's – now look as follows (notice the distracting borders):

![mushroom-new](https://user-images.githubusercontent.com/30414540/199621315-5015475a-4573-43cb-9618-e97af836be12.png)

### Expected Behavior

Mushroom's own cards (and built-in ones styled using Mushroom Themes) retain their expected design:

![mushroom-old](https://user-images.githubusercontent.com/30414540/199621300-37606bb1-a6b5-4aaf-88f1-b900a04aad53.png)

### Environment

```markdown
- Version: 2.3.1 (Mushroom), 0.0.6 (Mushroom Themes)
- HASS: 2022.11
```

### Anything else?

There is an entry in the recent [`2.2.0` changelog](https://github.com/piitaya/lovelace-mushroom/releases/tag/v2.2.0) regarding compatibility with HA 2022.11 ("_Compatible with 2022.11 HA version_"), but it appears as though this isn't fully functional yet. 🙂

**Comment by quadrupole1988** [comment]
I have the same issue after updating to 2022.11

**Comment by tobpet** [comment]
I have the same issue after updating to 2022.11

**Comment by piitaya** [comment]
Hello 🙂
The default Home Assistant theme has changed. Box-shadow has been replaced by borders. Mushroom follow these guidelines, box shadow has been dropped too.
If you don't like it (because it's just a personal preference), you can re-add it by adding theses lines to your theme :
```
--ha-card-box-shadow: 0px 2px 4px 0px rgba(0,0,0,0.16);
--ha-card-border-width: 0;
```

**Comment by tobpet** [comment]
Is there a way to make borders on chips disapear?

**Comment by Chronos47** [comment]
There must be an easy way to restore the previous way it worked. Not a developer way. I chose mushroom for the drop shadows.

**Comment by piitaya** [comment]
You can restore the old theme "Mushroom Shadow" with the [0.0.7 version](https://github.com/piitaya/lovelace-mushroom-themes/releases/tag/v0.0.7).

**Comment by Aohzan** [comment]
Even if we are ok with the new design, the `mushroom-chips-card` is still a problem, no ?
![image](https://user-images.githubusercontent.com/2736322/199688187-d1745e00-c787-49b7-90ac-14236621b648.png)

**Comment by piitaya** [comment]
@Aohzan You just have a cache issue, try restarting home assistant or empty your cache in your browser 🙂

**Comment by Aohzan** [comment]
indeed, thanks (ctrl+f5 didn't work, which I tried after each beta update :grin:, had to clean entire cache)

**Comment by CumpsD** [comment]
> You can restore the old theme "Mushroom Shadow" with the [0.0.7 version](https://github.com/piitaya/lovelace-mushroom-themes/releases/tag/v0.0.7).

Is there a Mushroom Square Shadow too? ;)

**Comment by BVollmerhaus** [comment]
I suppose this can be closed with the release of [Mushroom Themes `0.0.9`](https://github.com/piitaya/lovelace-mushroom-themes/releases/tag/v0.0.9). Thanks for the quick fix, @piitaya and @CumpsD.
